### PR TITLE
Add 4.4, 4.5 for openshift-tests-private

### PR DESCRIPTION
**Pull request by jianzhangbjz**
As the title show, add openshift-ci for the release-4.4, 4.5 branches.

### CODE REVIEW

**Comment by petr-muller** [comment]
> /assign @petr-muller

No, the configs should now be team administered. You should ask people from your OWNERS.

/unassign

**Comment by jianzhangbjz** [comment]
@petr-muller Could you help to merge it? Thanks!

**Comment by jianzhangbjz** [comment]
@petr-muller Thanks for your information! I assign it to myself since only myself maintains this repo now. It looks good to me, could you help to merge it? Thanks!

**Comment by petr-muller** [comment]
> @petr-muller Thanks for your information! I assign it to myself since only myself maintains this repo now. It looks good to me, could you help to merge it? Thanks!

Sorry, no. We are intentionally not involved in component configs, that's why we enforce component configs to have `OWNERS`. Relying on DPTP does not scale (especially for us), and repositories with a single maintainer are a maintenance anti-pattern. You should ask a team member (or more) to review CI config changes.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.5.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  namespace: ocp
+test_binary_build_commands: make test
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-test
+      commands: ./bin/extended-platform-tests run openshift/conformance/parallel --run
+        "High-24829"
```
Comment (shawn-hurley) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.5.yaml:47]: Does this mean that we are only testing a single test for 4.4 and 4.5?

Comment (jianzhangbjz) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.5.yaml:47]: @shawn-hurley Yes, this repo is private for the QE. But, the logs are public. We don't want to output any sensitive logs, so just add test one case here. And, here, we just want to make sure the generated binary `extended-platform-tests` works.

**Comment by jianzhangbjz** [comment]
@kuiwang02 Could you help to review it? Thanks!
/unassign

**Comment by jianzhangbjz** [comment]
@petr-muller Thanks! Got it. I have asked one member to review it.

**Pull request merged**